cmd/launcher/options.go

```diff
@@ -52,141 +50,60 @@ const (
+		flControlServerURL  = flagset.String("control_hostname", "", "The hostname of the control server")
+		flEnrollSecret      = flagset.String("enroll_secret", "", "The enroll secret that is used in your environment")
+		flEnrollSecretPath  = flagset.String("enroll_secret_path", "", "Optionally, the path to your enrollment secret")
+		flGetShellsInterval = flagset.Duration("control_get_shells_interval", 3*time.Second, "The interval at which the get shells request will be made")
+		flInitialRunner     = flagset.Bool("with_initial_runner", false, "Run differential queries from config ahead of scheduled interval.")
+		flKolideServerURL   = flagset.String("hostname", "", "The hostname of the gRPC server")
+		flLoggingInterval   = flagset.Duration("logging_interval", 60*time.Second, "The interval at which logs should be flushed to the server")
+		flOsquerydPath      = flagset.String("osqueryd_path", "", "Path to the osqueryd binary to use (Default: find osqueryd in $PATH)")
+		flRootDirectory     = flagset.String("root_directory", "", "The location of the local database, pidfiles, etc.")
+		flRootPEM           = flagset.String("root_pem", "", "Path to PEM file including root certificates to verify against")
+		flVersion           = flagset.Bool("version", false, "Print Launcher version and exit")
+		_                   = flagset.String("config", "", "config file ro parse options from (optional)")
 
 		// Autoupdate options
-		flAutoupdate = flag.Bool(
-			"autoupdate",
-			env.Bool("KOLIDE_LAUNCHER_AUTOUPDATE", false),
-			"Whether or not the osquery autoupdater is enabled (default: false)",
-		)
-		flNotaryServerURL = flag.String(
-			"notary_url",
-			env.String("KOLIDE_LAUNCHER_NOTARY_SERVER_URL", autoupdate.DefaultNotary),
-			"The Notary update server (default: https://notary.kolide.co)",
-		)
-		flMirrorURL = flag.String(
-			"mirror_url",
-			env.String("KOLIDE_LAUNCHER_MIRROR_SERVER_URL", autoupdate.DefaultMirror),
-			"The mirror server for autoupdates (default: https://dl.kolide.co)",
-		)
-		flAutoupdateInterval = flag.Duration(
-			"autoupdate_interval",
-			duration("KOLIDE_LAUNCHER_AUTOUPDATE_INTERVAL", 1*time.Hour),
-			"The interval to check for updates (default: once every hour)",
-		)
-		flUpdateChannel = flag.String(
-			"update_channel",
-			env.String("KOLIDE_LAUNCHER_UPDATE_CHANNEL", "stable"),
-			"The channel to pull updates from (options: stable, beta, nightly)",
-		)
+		flAutoupdate         = flagset.Bool("autoupdate", false, "Whether or not the osquery autoupdater is enabled (default: false)")
+		flNotaryServerURL    = flagset.String("notary_url", autoupdate.DefaultNotary, "The Notary update server (default: https://notary.kolide.co)")
+		flMirrorURL          = flagset.String("mirror_url", autoupdate.DefaultMirror, "The mirror server for autoupdates (default: https://dl.kolide.co)")
+		flAutoupdateInterval = flagset.Duration("autoupdate_interval", 1*time.Hour, "The interval to check for updates (default: once every hour)")
```

Comment:
perhaps we can punt on it for this PR, but AFAICT the only way to figure out the expected format for this value is to read the code.

Reply:
I say punt. This is mostly an advanced debug option, it's not usually displayed. And, if you get it wrong:
```
dover:launcher seph$ ./build/launcher -autoupdate_interval 5
invalid value "5" for flag -autoupdate_interval: time: missing unit in duration 5
```

Reply:
ah ok, good enough for now then.